app/views/users/show.html.erb

```diff
@@ -0,0 +1,98 @@
+
+<div class="container-fluid w-100 px-5 mb-5">
+  <h3 class="text-center ml-5 mt-4 text-white">
+    Event List <strong> <%= @user.name %> </strong> Created
+    <span class="btn btn-success ml-5 p-2">
+      <%= link_to '+ Create New Event', new_event_path, class: "text-white p-2 nav-link link-color-danger" %>
+    </span>
+
+  </h3>
+  <div class="row mb-5 p-0">
+    <div class="col-xl-12">
+      <table class="table">
+          <thead class="thead-dark">
+              <tr>
+                <th>Status</th>
+                <th>Description</th>
+                <th>Creator</th>
+                <th>Date</th>
+                <th>Location</th>
+                <th>Event Page</th>
+                <th>Delete</th>
+              </tr>
+          </thead>
+          <tbody>
+            <% @events.order(date: :desc).each do |event| %>
```

Comment:
- [ ] There should be NO application logic in the views nor helpers. Views are just for displaying data that became available to it from the controllers, it should not know anything about the events order. 

Yet, controllers gather, organize, filter data and make it ready for display in its corresponding view file. So we can move ordering in the controller.

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png)